action.yml

```
@@ -16,14 +16,14 @@ inputs:
     description: Path to the tool's results file to share with Pixeebot.
     required: false
   sonar-token:
-    description: Token for authenticating requests to SonarCloud.
+    description: Token for authenticating requests to Sonar.
     required: false
   sonar-component-key:
-    description: Key identifying the SonarCloud component to be analyzed.
+    description: Key identifying the Sonar component to be analyzed.
     required: false
-  sonar-api-url:
-    description: Base URL of the SonarCloud API.
-    default: https://sonarcloud.io/api
+  sonar-host:
```

I'm sorry, I messed up the requirements for this change. They should have specified that the input name be `sonar-host-url`. Sonar tooling always refers to this as a "host URL". I think this makes sense, because it's actually more than a hostname: it's also the scheme and optional port.

```suggestion
  sonar-host-url:
```